server.js

```diff
@@ -153,7 +156,7 @@ function Server (opts) {
       if (req.method === 'GET' && (req.url === '/stats' || req.url === '/stats.json')) {
         infoHashes.forEach(function (infoHash) {
           var peers = self.torrents[infoHash].peers
-          var keys = Object.keys(peers)
+          var keys = Object.keys(peers.cache)
```

Comment:
I would rather keep the LRU instance and stick with the module API.
This line is a workaround since there is no `getKeys` function in LRU yet but next I use `LRU.peek` https://github.com/feross/bittorrent-tracker/pull/160/files/7f98203d5ec63f395e2794700a10046c54dcf8f1#diff-78c12f5adc1848d13b1c6f07055d996eR172

We should not base all the function on an inner property of an external module, don't you think?